cmd/faucet/website.go

```diff
@@ -105,27 +105,27 @@ func Asset(name string) ([]byte, error) {
 }
 
 // AssetString returns the asset contents as a string (instead of a []byte).
-func AssetString(name string) (string, error) {
-	data, err := Asset(name)
-	return string(data), err
-}
+//func AssetString(name string) (string, error) {
+//	data, err := Asset(name)
+//	return string(data), err
+//}
```

Comment:
Linter was complaining about these not being called, but the top of the file says do not edit... this feels 😬 💀

Reply:
weird! I think not editing it makes sense... I wonder if the linter tests are passing on geth's master right now. 🤔

Reply:
[This](https://travis-ci.org/ethereum/go-ethereum/builds/651941757) is the build for the commit with the 1.9.11 label. Its failing for an unrelated reason. 🤷‍♂